pkg/hooks/hooks.go

```diff
@@ -21,6 +21,7 @@ package hooks
 
 import (
 	"encoding/json"
+	"os"
```

Comment:
Commit message: "virt-launcher, sidecars:"
The change is actually mostly in the VMI controller's template service.

nit: Please consider adding a link to k8s' `SubPath` documentation:
https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath